desktop/extensions-sdk/architecture/metadata.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Extension metadata
+description: Docker extension metadata
+keywords: Docker, extensions, sdk, metadata
+redirect_from:
+- /desktop/extensions-sdk/extensions/metadata
```

Comment:
I see the old URL was all caps;

```suggestion
- /desktop/extensions-sdk/extensions/METADATA/
```

Reply:
You are right! Solved in [81c885a](https://github.com/docker/docs/pull/16289/commits/81c885abcd39656199e843d95aaf486efc2a95c8) thanks!